src/lib/openjp2/t1.c

```diff
@@ -1786,6 +1786,34 @@ static void opj_t1_clbl_decode_processor(void* user_data, opj_tls_t* tls)
                 }
             }
 #endif
+
+#ifdef __loongarch_asx
+            {
+                asm volatile
+                (
+                    "srli.w        $t0,     %[cblk],   4      \n\t"
+                    "xvldrepl.w    $xr0,    %[step],   0      \n\t"
```

Comment:
Is there any reason not to use `xvreplgr2vr.w` here?

Reply:
Nice catch, will update.